evap/staff/templates/staff_questionnaire_form.html

```diff
@@ -82,6 +91,17 @@ <h5 class="card-title">{% trans 'Questions' %}</h5>
                 applySelect2(row.find("select"));
             };
             makeFormSortable("question_table", "questions", rowChanged, rowAdded, "", true, true);
+
+            // enable or disable checkbox for additional text answers
+            $('.question-type select').on('select2:select', function (e) {
+                const checkbox = $($(this).closest("td.question-type")).find("input");
+                if (e.params.data.id == 0 || e.params.data.id == 5) {  // 0: Text question; 5: Heading
+                    checkbox.prop("checked", false);
+                    checkbox.prop("disabled", true);
```

Comment:
shouldn't it be possible to disable the form fields in the form class?

Reply:
yes, the fields are disabled on loading the form. this here is to disable them when editing the form and choosing one of the options that don't allow additional text answers.